R/confirmatoryfactoranalysis.R

```diff
@@ -898,6 +941,38 @@ confirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
     vi$setData(vidat)
     vi$dependOn(optionsFromObject = jrobject)
   }
+
+
+  # Thresholds
```

Comment:
Could you put this code in a separate function? The idea is that we could reuse this `jaspSem` (or perhaps it would make more sense if this was implemented in `jaspSem` and `jaspFactor` just reused the function from there, since `jaspFactor` already depends on `jaspSem`).

Reply:
Good idea. Looking at the SEM tables, they do have different labels for the standardized columns. Also, are we sure the Thresholds table will always look the same for SEM and Factor?